Makefile

```diff
@@ -1,10 +1,24 @@
-.PHONY: generate localmkdocs test
+.PHONY: generate regenerate localmkdocs test
 
 LATEST := v10.0.0
 
-generate:
-	./scripts/generate.sh v10.0.0
-	./scripts/generate_latest.sh v10.0.0
+generate: gen/grafonnet-latest
+
+gen/grafonnet-latest:
+	./scripts/generate.sh ${LATEST} && \
+	./scripts/generate_latest.sh ${LATEST}
+
+regenerate: gen/grafonnet-v*
+
+# is latest, use `make generate` instead
+#gen/grafonnet-v10.0.0:
+#	./scripts/generate.sh v10.0.0
+
+gen/grafonnet-v9.5.0:
```

Comment:
Why 9.5.0 and not the latest patch? Same for all versions

Reply:
This builds on the grok pipeline, following the major releases: https://github.com/grafana/grok/tree/main/jsonnet